candidates/templates/candidates/posts.html

```diff
@@ -19,7 +19,7 @@
         <h3>{{ role_data.role }}</h3>
         {% for election_data in role_data.elections %}
           {% with election=election_data.election %}
-            <h4>{{ election.name }}</h4>
+            <a href="{% url 'constituencies' election=election.slug %}"><h4>{{ election.name }}</h4></a>
             <ul>
```

Comment:
I agree with this change, but it's interesting to me that it's actually valid HTML5 – previously a block level element couldn't be inside in inline element, but the HTML5 validates passes this.